### ISSUE TITLE
Better scoring system

### ISSUE THREAD

**Issue by heatherleaf** [MUSTE-Project/MULLE#41]
Currently, the scoring consists of:

- the number of changes -- i.e., the number of times the user really has selected a menu item
- the time in seconds

Is that the best scoring system? Possible alternatives:

- the number of times you show a menu
- something more complicated, e.g., `score += treediff(a,b)` every second

Related: do we want high-score or low-score? Currently a low score is good, but usually games go for high-scores

@daherb should have a say here too

**Comment by fredefox** [comment]
I have set this up so it should be relatively easy to implement now. We just need to decide on what metrics to collect.

**Comment by fredefox** [comment]
I'm putting this on-hold.

**Comment by fredefox** [comment]
Removing milestone since this is on hold.

**Comment by heatherleaf** [comment]
Just some random thoughts:

- every (good) move should give points
- but it should not be possible to go "back-and-forth" just to collect points
    - meaning that bad moves must give negative points
- the time should be a factor too

How do we know if a move is good (and how good)?

- simple: score = ∆ edit distance (between current sentence and goal)
- more difficult: score = ∆ best path (from current sentence to goal)
- (perhaps divided by the elapsed time?)

**Comment by heatherleaf** [comment]
Depends on #98
